simplq/src/components/pages/Status/LeaveQueue.jsx

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
+import CloseIcon from '@material-ui/icons/Close';
+import IconButton from '@material-ui/core/IconButton';
+import { useHistory } from 'react-router-dom';
 import styles from '../../../styles/statusPage.module.scss';
 
 export default () => {
+  const history = useHistory();
+
   return (
-    <>
-      <div className={styles['user-action']}>
-        {/* icon */}
-        <div>
-          <h2>Leave Queue</h2>
-          <p>Exit from the queue</p>
-        </div>
+    <li className={styles['user-action']}>
+      <IconButton onClick={() => history.push(`/`)} aria-label="Leave Queue">
```

Comment:
Can we make the entire `<li>` element clickable instead of just the icon? 

We should then change to pointer to hand for `user-action`, and add onHover change background colour to `#f8f9fa`.

Reply:
I did it and it works, but I run now into these problems with eslint:

  ```
14:5  error  Visible, non-interactive elements with click handlers must have at least one keyboard listener  jsx-a11y/click-events-have-key-events
  14:9  error  Non-interactive elements should not be assigned interactive roles                               jsx-a11y/no-noninteractive-element-to-interactive-role
```

The first one is fixable, the second not. I could turn the whole `li` into a `button` and remove all the styles from it.

Reply:
For one let's add a keydown?

Is it possible to replace the li with an anchor or button and style it appropriately? Will this same strategy work for the expansion panel below when we implement that?

The idea is, once you finish this, we should refactor and move this out into a common place and reuse this as a "SidePanel" component here and in the admin page.

Reply:
Making it a button is fine imo, as long as the styles match the design.

Reply:
And the sidepanel common component is already made. After pulling the latest changes, you should see it too @Gh05d.

Reply:
I went with the _naked_ button solution as it is the easiest way and also the semantically correct one as screen readers will pick it up correctly too.